### ISSUE TITLE
Example mMR geometry may be out of date?

### ISSUE THREAD

**Issue by robbietuk** [UCL/STIR-GATE-Connection#88]
https://github.com/UCL/STIR-GATE-Connection/blob/master/ExampleScanners/mMR/geometry_mMR.mac

This example mMR configuration does not use the virtual crystals found in the mMR. However, STIR can now support these virtual crystals (https://github.com/UCL/STIR/pull/833 etc.).

**Comment by KrisThielemans** [comment]
> STIR can now support these virtual crystals

That is correct. https://github.com/UCL/STIR/blob/6417022e54a4814f25776bc8ca1df3acd4e865f6/src/buildblock/Scanner.cxx#L1620-L1630 hardwires 1 virtual crystal for the mMR (and som eother scanners). This can currently not be set via the interfile header. We do have keywords in the `.hroot` file
```
Number of virtual axial crystals per block := ...
Number of virtual transaxial crystals per block:= ...
```
However, currently they [only check consistency](https://github.com/UCL/STIR/blob/6417022e54a4814f25776bc8ca1df3acd4e865f6/src/buildblock/Scanner.cxx#L1646-L1651)

So, to use the (correct) 8x8 block, the following would have to be updated (e.g. by setting `originating system` to `Siemens mMR`)
https://github.com/UCL/STIR-GATE-Connection/blob/c5ac1e09ceda3efd6a5ec67c3b395f94b0d1c138/ExampleScanners/mMR/root_header_template.hroot#L7
https://github.com/UCL/STIR-GATE-Connection/blob/c5ac1e09ceda3efd6a5ec67c3b395f94b0d1c138/ExampleScanners/mMR/root_header_template.hroot#L31
https://github.com/UCL/STIR-GATE-Connection/blob/c5ac1e09ceda3efd6a5ec67c3b395f94b0d1c138/ExampleScanners/mMR/STIR_scanner.hs#L4
https://github.com/UCL/STIR-GATE-Connection/blob/c5ac1e09ceda3efd6a5ec67c3b395f94b0d1c138/ExampleScanners/mMR/STIR_scanner.hs#L40
https://github.com/UCL/STIR-GATE-Connection/blob/c5ac1e09ceda3efd6a5ec67c3b395f94b0d1c138/ExampleScanners/mMR/STIR_scanner.hs#L43
